EasyReflectometry/sample/elements/layers/layer_apm.py

```diff
@@ -103,7 +103,7 @@ def __init__(
         :param molecular_formula: Formula for the molecule in the layer.
         :param thickness: Layer thickness in Angstrom.
         :param solvent: Solvent containing the molecule.
-        :param solvent_surface_coverage: Fraction of layer not covered by molecule.
+        :param solvation: Fraction of layer not covered by molecule.
```

Comment:
"Fraction of layer not covered by molecule." --> "Fraction of layer not covered by surfactant layer"